devices/Bmm150/app.config

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
please remove all app.config. They are creating issues when it comes to updating versions

Reply:
Removed.